packages/blockchain-wallet-v4-frontend/src/components/Flyout/InterestUploadDocuments/GetStarted.tsx

```diff
@@ -0,0 +1,158 @@
+
+const TextContentWrapper = styled.div`
+  padding: 0 40px;
+  display: flex;
+  flex-direction: column;
+`
+
+const Items = styled.div`
+  margin-top: 44px;
+`
+
+const Item = styled.div`
+  padding: 20px 40px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`
+
+const TierDescription = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 0 10px;
+`
+
+const TierTitle = styled(Text)`
+  color: ${(props) => props.theme.grey900};
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 150%;
+`
+
+const ItemSubtitle = styled(Text)`
+  color: ${(props) => props.theme.grey900};
+  font-size: 16px;
+  font-weight: 500;
+  margin-bottom: 4px;
+`
+
+const GetStarted = ({ nextStep, previousStep }: Props) => {
+  return (
+    <Container>
+      <Header data-e2e='InterestUploadDocumentsCloseButton' mode='back' onClick={previousStep}>
+        <FormattedMessage
+          id='modals.interest.withdrawal.upload_documents.get_started.title'
+          defaultMessage='Upload & Verify'
+        />
+      </Header>
+      <Content mode='top'>
+        <TextContentWrapper>
+          <Text size='14px' weight={500} color='grey900' lineHeight='20px'>
+            <>
+              <FormattedMessage
+                id='modals.interest.withdrawal.upload_documents.get_started.description'
+                defaultMessage='Please upload a clear photo of each file or document listed below.'
+              />{' '}
+              <Link
+                size='16px'
+                weight={500}
+                target='_blank'
+                href='https://support.blockchain.com/hc/en-us/'
+              >
+                <FormattedMessage id='buttons.learn_more_arrow' defaultMessage='Learn more ->' />
+              </Link>
+            </>
+          </Text>
+        </TextContentWrapper>
+
+        <Items>
+          <Item>
+            <div>
+              <Image name='proof-of-address' size='32px' />
+            </div>
+            <TierDescription>
+              <TierTitle>
+                <FormattedMessage
+                  id='modals.interest.withdrawal.upload_documents.get_started.proof_of_current_address'
+                  defaultMessage='Proof of Current Address'
+                />
+              </TierTitle>
+              <ItemSubtitle color='grey900' size='14px' weight={500}>
+                <FormattedMessage
+                  id='modals.interest.withdrawal.upload_documents.get_started.bank_acccount_statement'
+                  defaultMessage='Bank Account Statement or Utility Bill'
+                />
+              </ItemSubtitle>
+
+              <Text color='grey600' lineHeight='1.5' size='12px' weight={500}>
+                <FormattedMessage
+                  id='modals.interest.withdrawal.upload_documents.get_started.bank_acccount_statement.description'
+                  defaultMessage='Must be dated within the last 90 days or a council tax bill for the current tax year.'
```

Comment:
What is a "council" tax bill? I've never heard of it, maybe it's supposed to be a different word?

Reply:
Will double check with product since I just used labels from Figma